docs/advanced/advanced_troubleshooting.md

```diff
@@ -0,0 +1,124 @@
+# Troubleshooting
+Use this section for advanced troubleshooting information for your CSI driver.
+
+- [Verifying the CSI driver is running](#Verifying-the-CSI-driver-is-running)
+- [Multipath troubleshooting](#Multipath-troubleshooting)
+- [General troubleshooting](#General-troubleshooting)
+
+## Verifying the CSI driver is running
+
+  Verify that the CSI driver is running. (Make sure that all components are in the  _Running_ state), using the `$> kubectl get all -n <namespace> -l csi` command.
+
+## Multipath troubleshooting
+
+Use this information to help pinpoint potential causes for multipath failures.
+
+**Note:** Run these commands should be run on one of the worker nodes.
+
+-   **Display multipath information (FC and iSCSI)**
+
+    Display multipath information, using the `sudo multipath -ll` command.
+
+    <pre>
+    mpathb (3600507680283851530000000000000a6) dm-0 IBM,2145
+    size=1.0G features='1 queue_if_no_path' hwhandler='1 alua' wp=rw
+    |-+- policy='service-time 0' prio=50 status=active
+    | `- 3:0:0:0 sda 8:0  active ready running
+    `-+- policy='service-time 0' prio=10 status=enabled
+    `- 2:0:0:0 sdb 8:16 active ready running
+
```

Comment:
is the blank line at the end of the block on purpose?

Reply:
The line is for clarity in the writing and the parts. It's not an extra line in the block itself

Reply:
![image](https://user-images.githubusercontent.com/57443811/149777080-7c08f813-7273-4146-af42-561044f72406.png)
are you sure?

Reply:
DONE - see https://github.com/IBM/ibm-block-csi-driver/pull/453
i think i fixed this now